backend/src/db/schemas/super-admin.ts

```diff
@@ -16,7 +16,8 @@ export const SuperAdminSchema = z.object({
   allowedSignUpDomain: z.string().nullable().optional(),
   instanceId: z.string().uuid().default("00000000-0000-0000-0000-000000000000"),
   trustSamlEmails: z.boolean().default(false).nullable().optional(),
-  trustLdapEmails: z.boolean().default(false).nullable().optional()
+  trustLdapEmails: z.boolean().default(false).nullable().optional(),
+  defaultOrgId: z.string().uuid().nullable().optional()
```

Comment:
what about `defaultAuthOrg`? Incase we have default org for something else in the future

Reply:
This name is a bit misleading to me. What is the default org to is my question that keeps popping?

Reply:
Resolved! 

`defaultOrgId` is now called `defaultAuthOrgId` and `defaultOrgSlug` is called `defaultAuthOrgSlug`.